core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
Is this only for development? I guess performance improvement of mac isn't very important in production, so maybe we can have Gradle dynamically add this dependency so it doesn't affect the jar.